doc/source/development/community.rst

```diff
@@ -0,0 +1,107 @@
+.. _community:
+
+=====================
+Contributor community
+=====================
+
+pandas is a community-driven open source project developed by a large group
+of `contributors <https://github.com/pandas-dev/pandas/graphs/contributors>`_
+and a smaller group of `maintainers <https://pandas.pydata.org/about/team.html>`_.
+The pandas leadership has made a strong commitment to creating an open,
+inclusive, and positive community. Please read the pandas `Code of Conduct
+<https://pandas.pydata.org/community/coc.html>`_ for guidance on how to
+interact with others in a way that makes the community thrive.
+
+We offer several meetings and communication channels to share knowledge and
+connect with others within the pandas community.
+
+Community meeting
+-----------------
```

Comment:
Could you add line breaks after every header? Sorry, probably was not very clear earlier

Reply:
oh ofc, I'm on it.